letsencrypt/domains.sls

```diff
@@ -42,14 +42,14 @@
 create-initial-cert-{{ setname }}-{{ domainlist | join('+') }}:
   cmd.run:
     - unless: {{ check_cert_cmd }} {{ setname }}
-    - name: {{ create_cert_cmd }} certonly --quiet --cert-name {{ setname }} -d {{ domainlist|join(' -d ') }} --non-interactive
+    - name: {{ create_cert_cmd }} {{ letsencrypt.create_init_cert_subcmd }} --quiet --cert-name {{ setname }} -d {{ domainlist|join(' -d ') }} --non-interactive
```

Comment:
It seems there is no default value for `create_init_cert_subcmd`, what does happen here if it's not set in pillar?

Reply:
@daks The default is supplied but needs to be fixed, as mentioned above:

* https://github.com/saltstack-formulas/letsencrypt-formula/pull/77/files#diff-85fe05632fbab67e7af1ce552c2237b89d2fe6515d769a76639f85c62c64a745R34

Reply:
fixed